DEVELOPMENT.md

```diff
@@ -119,6 +119,14 @@ Disable the following inspections:
 - ``Java | Performance | Call to 'Arrays.asList()' with too few arguments``,
```

Comment:
Commit message:
 - typo `Byt`
 - `Byt since we're working with SQL statements a lot, I find it useful.` maybe let's remove this statement. It does not add much value to the commit.